src/pdfkit.cs

```diff
@@ -26,8 +26,25 @@
 //
 // PdfKit.cs: Bindings for the PdfKit API
 //
+#if MONOMAC || (IOS && XAMCORE_2_0)
+
 using System;
+#if MONOMAC
 using XamCore.AppKit;
+#else
+using XamCore.UIKit;
+using NSColor = XamCore.UIKit.UIColor;
+using NSFont = XamCore.UIKit.UIFont;
+using NSImage = XamCore.UIKit.UIImage;
+using NSView = XamCore.UIKit.UIView;
+using NSEdgeInsets = XamCore.UIKit.UIEdgeInsets;
+using NSEvent = XamCore.UIKit.UIEvent;
+using NSBezierPath = XamCore.UIKit.UIBezierPath;
+using NSTextAlignment = XamCore.UIKit.UITextAlignment;
+// HACK: to make intermediate dll build
+using NSPrintInfo = XamCore.Foundation.NSObject;
```

Comment:
Yes, we use this pattern in other bindings, since we'll use a `[NoiOS]` on the definition that uses `NSPrintInfo` we'll be fine and avoid `#if MONOMAC/IOS` inside the file